package.json

```diff
@@ -1,14 +1,12 @@
 {
   "private": true,
   "name": "prism",
-  "license": "UNLICENSED",
-  "version": "0.0.0",
+  "license": "Apache-2.0",
```

Comment:
Why Apache-2.0? Have that been confirmed with @marbemac / @brianmrock ?
I know little about licenses, so maybe that's good but I remember Marc or Brian said that we'll need to be careful with this.

Reply:
I've asked @philsturgeon — and he mandated to use the same in Spectral, which is Apache 2.

BTW in case you're curious about the differences: https://tldrlegal.com

This was super beneficial to me

Reply:
@XVincentX good link definitely clears up a lot, thanks! Still I don't have opinion, I will let someone else approve that part.